setup.py

```diff
@@ -97,7 +97,7 @@ def _collect_requirements(requires):
         "sphinxcontrib-mermaid~=0.7.1",
         "myst-parser~=1.0.0",
         "Jinja2<3.1.0",
-        "kedro-datasets[all]~=1.4.2",
+        "kedro-datasets[all]==1.5.0",
```

Comment:
Oversight:

https://github.com/kedro-org/kedro-plugins/blob/3a3cffb271057cac0142ed202395704bfcf96b2c/kedro-datasets/pyproject.toml#L38-L51

`pandas.DeltaTableDataSet` is not included in `[pandas]`, and also

https://github.com/kedro-org/kedro-plugins/blob/3a3cffb271057cac0142ed202395704bfcf96b2c/kedro-datasets/pyproject.toml#L126-L133

`pandas` is not included in `[all]`.

Hence `deltalake` doesn't get installed, hence `pandas.DeltaTableDataSet` import fails.

Workaround that doesn't need a new `kedro-datasets` release:

```suggestion
        "kedro-datasets[all,pandas-deltatabledataset]~=1.5.0",
```

Reply:
Ah that makes sense I was confused why it wasn't building, thank you @astrojuanlu!